code/game/machinery/barsigns.dm

```diff
@@ -440,6 +440,12 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/barsign, 32)
 	desc = "First stop out of hell, last stop before heaven."
 	neon_color = COLOR_RED
 
+/datum/barsign/orangejuice
+	name = "Oranges' Juicery"
```

Comment:
Pretty sure we've tried to avoid referencing orange, and this could be seen as one

Reply:
sure, changed